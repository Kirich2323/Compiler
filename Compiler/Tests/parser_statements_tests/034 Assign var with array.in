var
    a : array [1..10] of integer;
    b : integer;
begin
    b := a[1];
end.