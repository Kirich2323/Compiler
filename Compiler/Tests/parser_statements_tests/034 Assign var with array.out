\- main block
   \- :=
      |- b
      \- []
         |- a
         \- 1